.redhat-ci.yml

```diff
@@ -0,0 +1,25 @@
+branches:
+    - master
+    - auto
+    - try
+
+container:
+    image: jlebon/ostree-tester:rhci
```

Comment:
The problem with this then is that if it's only updated manually, it's going to bitrot...

At some point of course, we should implement `rpm-ostree assemble-buildroot` or something, which would be fast *and* accurate.